From c6c79512e357276c27ce08d0e95b48a503ebac1c Mon Sep 17 00:00:00 2001
From: Matthew Schinckel <matt@schinckel.net>
Date: Fri, 25 May 2018 21:04:03 +0930
Subject: [PATCH] Ensure we use a valid AR.

---
 ota-server/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ota-server/Makefile b/ota-server/Makefile
index f05f346..3f8174b 100644
--- a/ota-server/Makefile
+++ b/ota-server/Makefile
@@ -1,5 +1,6 @@
 CC = xtensa-lx106-elf-gcc
 LD = xtensa-lx106-elf-ld
+AR = xtensa-lx106-elf-ar
 ESP_SDK = $(shell $(CC) -print-sysroot)/usr
 CFLAGS = -std=gnu99 -Os -I. -mtext-section-literals -mlongcalls -mforce-l32 -ffunction-sections -fdata-sections -DLWIP_OPEN_SRC -D__ets__
 LDLIBS = -L$(ESP_SDK)/lib -L. -laxtls -lmain -lnet80211 -lwpa -llwip_open -lpp -lphy -lnet80211 -lgcc
